lib/pages/community_page/community_page.dart

```diff
@@ -32,14 +38,63 @@ class CommunityPageState extends State<CommunityPage> {
   bool _isUpdatingMembership = false;
   bool _isloading = true;
 
+  final String _currentDate = DateFormat("dd/MM/yyyy").format(DateTime.now());
```

Comment:
Yes so on the server side the date is also generated in Est time zone, In production code it should be formatted for one specific time zone but since this app is only going to be used by us it's not necessary to specify the time zone here.